CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
+
+## HEAD
+
+* Replace me with one change per line
+* Please :)
```

Comment:
We've used the Github Release view as a changelog before. Should we have this refer to it instead of keeping two sources in sync?